CHANGELOG.md

```diff
@@ -20,14 +20,14 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Features
 
-* **detector:** skip dublicates in same folder if `--skipLocal` used ([7029ddd](https://github.com/kucherenko/jscpd/commit/7029ddd)), closes [#326](https://github.com/kucherenko/jscpd/issues/326)
+* **detector:** skip duplicates in same folder if `--skipLocal` used ([7029ddd](https://github.com/kucherenko/jscpd/commit/7029ddd)), closes [#326](https://github.com/kucherenko/jscpd/issues/326)
```

Comment:
some projects don't like to change changelogs (I understand that the original text was in a commit message, but the people reading the commits can still benefit from being able to understand the text w/o typos, e.g. if they're reading it using a translation service) -- I'm happy to drop this or anything else.